src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -48,6 +61,47 @@ public PythonEngineVersion Engine
             }
         }
 
+        private ObservableCollection<PythonEngineVersion> availableEngines;
+        /// <summary>
+        /// Available Python engines.
+        /// </summary>
+        public ObservableCollection<PythonEngineVersion> AvailableEngines
```

Comment:
same question about auto properties

Reply:
This is for the purpose of initializing it only once.

Reply:
I guess I'm still a bit confused by this answer - 
can't you do `Public ObserverableCollection<PythonEngineVersion> {get; set;} = ObserverableCollection<PythonEngineVersion>(){ yada yada}` ?

Reply:
Yes, you are right. I guess I don't really like to initialize in that way when it takes more than one line. If it was just creating the collection I wouldn't mind.

Reply:
Oh, I see you put `{ yada yada}` in same line. I don't really like that either 😄